.travis.yml

```diff
@@ -17,13 +22,19 @@ cache:
 before_script:
   - curl -sSL https://github.com/maidsafe/QA/raw/master/travis/cargo_install.sh > cargo_install.sh
   - bash cargo_install.sh cargo-prune;
-  - rustup component add rustfmt
-  - rustup component add clippy
+  - rustup target add $TARGET
+  - rustup component add rustfmt clippy
+  - if [ "${TRAVIS_OS_NAME}" = osx ]; then
+      mkdir -p .cargo;
+      echo "[target.$TARGET]" >> .cargo/config;
```

Comment:
This config file needs a newline at the end.  Also, using `printf` rather than `echo` would allow this to go on one line with `\n`(s) used I believe.
Having said all that, @m-cat's suggestion is probably better.